src/stores/userStore.ts

```diff
@@ -0,0 +1,48 @@
+import { useAuth0 } from '@auth0/auth0-vue'
+import { acceptHMRUpdate, defineStore } from 'pinia'
+import { computed, ref } from 'vue'
+import { PiniaPersistedStateOptions } from '~/utils/data'
+
+export const useUserStore = defineStore('user', () => {
+  const { isAuthenticated, loginWithRedirect, logout, user } = useAuth0()
+
+  const isCookieAccepted = ref(false)
+  const cookiePopupVisible = ref(true)
+
+  const isUserAuthenticated = computed(() => {
+    return isAuthenticated.value
+  })
+  const currentUser = computed(() => {
+    return user.value
+  })
+
+  const handleLogin = () => {
+    loginWithRedirect({ appState: { targetUrl: '/' } })
```

Comment:
Does this mean, I'll always end up at the home-page after I logged in or is this setting for something different ..?